pkg/aggregator/aggregator.go

```diff
@@ -704,3 +713,19 @@ func (agg *BufferedAggregator) run() {
 		}
 	}
 }
+
+// tags returns the list of tags that should be added to the agent telemetry metrics
```

Comment:
Might be worth commenting here that the container agent tags may be missing in the first seconds after agent startup